source/scripts/tests/TrollTest.js

```diff
@@ -0,0 +1,52 @@
+import Expect from "expect.js"
+
+import Game from "scripts/model/Game.js"
+
+import MONSTERS from "scripts/data/monsters.js"
+
+import DATA from "scripts/data/index.js"
+
+export default function TrollTest() {
+
+    // We are a testing a game.
+
+    var game = new Game({
+
+        // A game with an adventurer.
+        adventurer: {
+            position: {x: 3, y: 3},
+        },
+
+        // A game with an adventurer and a troll.
+
+        monsters: [
+            {
+                position: {x: 1, y: 1},
+                protomonster: MONSTERS.WHITE_TROLL
+            }
+        ]
+
+        // Also, by purposefully forgetting to
+        // pass a "wave" to the game, our test
+        // won't be interrupted by other monsters
+        // being spawned in.
```

Comment:
I put these comments in here for your edification; please delete them after you're done.